assets/scss/jotw-supplements.scss

```diff
@@ -0,0 +1,18 @@
+.jotw-secondary-text {
+  color:$govuk-secondary-text-colour;
+}
+
+@media (max-width: 40.0525em) {
+  .jotw-width_hide-narrow-40 {
+    display: none;
+  }
+}
+@media (min-width: 40.0525em) {
+  .jotw-width_hide-narrow-40 {
+    display: none;
+  }
+}
+
+.jotw-banner__button {
+  float: right;
+}
```

Comment:
No ending newline. This can impact some older systems where the empty line at the end of a file indicates the true end of file.

Reply:
https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file#answer-5813359

Reply:
Noted, will tidy up tomorrow.

Reply:
Done, I think I got all the SCSS files that were missing blank end-lines.